components/21-atoms/link/_link.scss

```diff
@@ -150,6 +150,20 @@ a {
       padding-left: .4rem;
     }
   }
+
+  &.dark-background {
```

Comment:
De dark-background class zou op het parent element moeten staan van de link. Zoals
```
<div class="dark-background">
  <a href="#">Link</a>
</div>
```

Je kan in de sass hiervoor op deze plaats (in het a element) het volgende gebruiken: `.dark-background & { ... }`. Zie bijv. https://codepen.io/Quillor/pen/wXBXqJ

Op de link zelf willen we geen extra classes zetten.

Reply:
Ik heb de code aangepast zodat er default een div rond staat met de class "light-background" en als er een background variabele meegeven wordt dan wordt deze gebruikt, heb dan de css nog aangepast zodat het werkt met de verandering en even getest. Alles werkt nog ook op de in the spotlight component.